README.md

```diff
@@ -259,6 +261,18 @@ const result2 = await otherBuilder
   .spawn();
 ```
 
+You can also register your own custom commands using the `registerCommand` or `registerCommands` methods:
+
+```ts
+const commandBuilder = new CommandBuilder()
+  .registerCommand("true" => () => { kind: "continue", code: 0 });
+
+const result = await commandBuilder
+  // now includes the 'true' command
+  .command("true && echo yay")
```

Comment:
It would be nice to build in `true` and `false` by the way :)

Reply:
I _knew_ you were going to say that :)

I have a few more tasks yet to work through, so, I imagine there will be a PR or two coming with more commands.  I'll put those on my list :)